weave-js/src/components/PagePanelComponents/Home/Browse3/pages/wfReactInterface/tsDataModelHooks.ts

```diff
@@ -270,11 +279,29 @@ const useCalls = (
       return {
         loading: false,
         result,
+        refetch,
       };
     }
-  }, [callRes, entity, project, opts?.skip]);
+  }, [callRes, entity, project, opts?.skip, refetch]);
 };
 
+const useCallsDelete = () => {
+  const getTsClient = useGetTraceServerClientContext();
+
+  const callsDelete = useCallback(
+    (projectID, ids): Promise<traceServerClient.TraceCallsDeleteRes> => {
+      return getTsClient()
+        .callsDelete({project_id: projectID, ids})
+        .catch(e => {
+          console.error(e);
```

Comment:
how do we want to handle errors? pass on to user?

Reply:
I think passing the Promise error to the caller is the best thing here for now.

Reply:
basically just don't catch and you are good